sources/packages/api/src/route-controllers/note/models/note.dto.ts

```diff
@@ -1,16 +1,67 @@
+import { IsIn, IsNotEmpty, MaxLength } from "class-validator";
+import {
+  Note,
+  NOTE_DESCRIPTION_MAX_LENGTH,
+  User,
+} from "../../../database/entities";
+
 /**
  * Base DTO for note.
  */
-export interface NoteBaseDTO {
+export class NoteAPIInDTO {
+  @IsNotEmpty()
   noteType: string;
+  @IsNotEmpty()
+  @IsIn([
```

Comment:
I believe that the decorator was applied to the wrong property. Besides that we have the enum `NoteType`, can we just use it here with the `@IsEnum(NoteType)`.